### ISSUE TITLE
Image Reconstruction Implementation

### ISSUE THREAD

**Issue by judydnguyen** [JeremyCJM/CCST#8]
Dear Authors,

Thanks for your great paper and public source code. I have a little concern about the experiments with image reconstruction experiments. I wonder why in the file https://github.com/JeremyCJM/CCST/blob/main/reconstruct_img/imagenet_reconstruct.py, we don't use any style statistic (style stats) of a specific domain. The loss function is the MSE loss between the reconstructed image and its corresponding style (not using the statistic information of the client -- a domain). I would really appreciate it if you could provide better guidance on this experiment. Thanks!

**Comment by JeremyCJM** [comment]
Hi Judy, it is because we need ground truth (GT) image to compute reconstruction loss during training, however, the average statistics do not have a corresponding GT. Therefore, the natural way is to train on image and image feature stats pairs, and then during inference time, you can test if you can reconstruct something meaningful from an average stat (domain style).

**Comment by judydnguyen** [comment]
Thank you for getting back to me. Yeah -- it makes sense. It will take more effort to design the inference attack using the clients' stats. I will mark it as close.

**Comment by chenrxi** [comment]
> Thank you for getting back to me. Yeah -- it makes sense. It will take more effort to design the inference attack using the clients' stats. I will mark it as close.

Hi Judy, Are you working on the inference attack using the clients' stats? Are there any corresponding manuscripts on this topic?
